### PR TITLE
fixed user error return

### PR DESCRIPTION
**Pull request by gianmarcomennecozzi**

### CODE REVIEW

**Comment by mrturkmen06** [comment]
@gianmarcomennecozzi 
Please prefer to use `hotfix`and `feature` prefix in front of branch name.

File: daemon/user.go [hunk 1]
```diff
@@ -143,7 +143,7 @@ func (d *daemon) DestroyUser(ctx context.Context, request *pb.DestroyUserRequest
 
 	if err := d.users.DeleteUserByUsername(request.Username); err != nil {
 		log.Error().Msgf("User delete error %v", err)
-		return &pb.DestroyUserResponse{Message: fmt.Sprintf("Error on deleting user %s, error: %v", request.Username, err)}, nil
+		return &pb.DestroyUserResponse{Message: fmt.Sprintf("Error on deleting user %s, error: %v", request.Username, err)}, err
```
Comment (mrturkmen06) [inline daemon/user.go:146]: What is the reason of changing this ?

Comment (gianmarcomennecozzi) [reply daemon/user.go:146]: Because we have to send back an error to the client since we got an error if its inside that `if` statement

**Comment by mrturkmen06** [comment]
I think, we can merge it once branch name is changed :) @gianmarcomennecozzi

**Comment by gianmarcomennecozzi** [comment]
you can merge both

**Pull request closed without merging**